src/Compilers/Core/MSBuildTask/Desktop/project.json

```diff
@@ -1,5 +1,7 @@
 {
-  "dependencies": { },
+  "dependencies": {
+    "Roslyn.Microsoft.Build": "0.0.2"
```

Comment:
https://www.nuget.org/packages/Roslyn.Microsoft.Build/ - I think packages that KevinR uploaded have Roslyn prefix, but the rest have RoslynDependencies.